src/test/java/teammates/test/cases/common/TimeHelperTest.java

```diff
@@ -82,6 +82,40 @@ public void testCombineDateTime() {
     }
     
     @Test
+    public void testIsWithinHour() {
+        Calendar timeCalendar = Calendar.getInstance(TimeZone.getTimeZone("UTC"));
+        
+        ______TS("Time within past hour");
+        
+        timeCalendar.add(Calendar.MINUTE, -10);
+        assertTrue(TimeHelper.isWithinHour(timeCalendar.getTime(),
+                Calendar.getInstance(TimeZone.getTimeZone("UTC")).getTime()));
+
+        ______TS("End time not within past hour");
+        
+        timeCalendar = Calendar.getInstance(TimeZone.getTimeZone("UTC"));
+        timeCalendar.add(Calendar.MINUTE, -70);
+        assertFalse(TimeHelper.isWithinHour(timeCalendar.getTime(),
+                Calendar.getInstance(TimeZone.getTimeZone("UTC")).getTime()));
+        
+        timeCalendar = Calendar.getInstance(TimeZone.getTimeZone("UTC"));
+        timeCalendar.add(Calendar.MINUTE, 10);
+        assertFalse(TimeHelper.isWithinHour(timeCalendar.getTime(),
+                Calendar.getInstance(TimeZone.getTimeZone("UTC")).getTime()));
+        
+        timeCalendar = Calendar.getInstance(TimeZone.getTimeZone("UTC"));
+        timeCalendar.add(Calendar.MINUTE, 10);
+        assertFalse(TimeHelper.isWithinHour(timeCalendar.getTime(), timeCalendar.getTime()));
+
+        ______TS("Session ended but grace time left");
+        int gracePeriod = 15;
+        timeCalendar = Calendar.getInstance(TimeZone.getTimeZone("UTC"));
+        timeCalendar.add(Calendar.MINUTE, -10);
+        assertFalse(TimeHelper.isWithinHour(new Date(timeCalendar.getTime().getTime() + gracePeriod * 60000),
+                Calendar.getInstance(TimeZone.getTimeZone("UTC")).getTime()));
```

Comment:
For this and many instances above this line, align `[C]alendar` with `[T]imeHelper`.